src/smartHandler.ts

```diff
@@ -167,8 +167,10 @@ export class SMARTHandler implements Authorization {
 
         if (fhirUserClaim && usableScopes.some((scope) => scope.startsWith('user/'))) {
             userIdentity.fhirUserObject = getFhirUser(fhirUserClaim);
-            userIdentity.patientOrgs = [];
```

Comment:
I am not 100 % sure but I think this initialization of userIdentity.patientOrgs = []; is required. Else there would be some errors in setting this value with patientOrgClaims. Could you please do some API call test with user as a Practitioner to verify this?

Reply:
Line 174 this file: `userIdentity.patientOrgs = patientOrgsClaim.map()` would create a list. So, initialization is not required.

yeah, I would add some unit tests around it